src/app/provider/view.nim

```diff
@@ -8,7 +8,7 @@ import chronicles
 import nbaser
 from base32 import nil
 
-const AUTH_METHODS = toHashSet(["eth_accounts", "eth_coinbase", "eth_sendTransaction", "eth_sign", "keycard_signTypedData", "eth_signTypedData", "personal_sign", "personal_ecRecover"])
+const AUTH_METHODS = toHashSet(["eth_accounts", "eth_coinbase", "eth_sign", "keycard_signTypedData", "eth_signTypedData", "personal_sign", "personal_ecRecover"])
```

Comment:
please add back eth_sendTransactions to AUTH_METHODS since this const is used to verify if the user has approved the `Web3` permission  (see here: https://github.com/status-im/nim-status-client/pull/1089/files#diff-df3e86487988b0340686a17c50011e91R102)
```suggestion
const AUTH_METHODS = toHashSet(["eth_accounts", "eth_coinbase", "eth_sendTransaction",  "eth_sign", "keycard_signTypedData", "eth_signTypedData", "personal_sign", "personal_ecRecover"])
```